app/views/layouts/application.html.erb

```diff
@@ -33,6 +33,66 @@
       </div>
     </div>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
+   
```

Comment:
Could the following be put in a separate javascript file under `app/javascript`?

Reply:
Sure, done.